bigtable-client-core-parent/bigtable-client-core/pom.xml

```diff
@@ -273,30 +266,24 @@ this is not used directly, but upgrading due to transitive vulnerabilities in ol
     </resources>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
+        <!-- skip for bigtable-client-core to allow for upcoming refactoring-->
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>clirr-maven-plugin</artifactId>
```

Comment:
was anything actually failing here? Or are we just doing this to be consistent?

Reply:
just consistency

Reply:
should we remove it? Feels like a good check to keep